plugins/woocommerce/src/Admin/Features/ProductBlockEditor/Init.php

```diff
@@ -720,6 +720,11 @@ public function set_current_screen_to_block_editor_if_wc_admin() {
 		// not be the product edit page (it mostly likely isn't).
 		if ( PageController::is_admin_page() ) {
 			$screen->is_block_editor( true );
+
+			wp_add_inline_script(
+				'wp-blocks',
+				'wp.blocks.unstable__bootstrapServerSideBlockDefinitions(' . wp_json_encode( get_block_editor_server_block_settings() ) . ');'
```

Comment:
Maybe we can add some protection in case `unstable__bootstrapServerSideBlockDefinitions` doesn't exist? Wouldn't that cause bigger issues?

```suggestion
				'wp.blocks.unstable__bootstrapServerSideBlockDefinitions && wp.blocks.unstable__bootstrapServerSideBlockDefinitions(' . wp_json_encode( get_block_editor_server_block_settings() ) . ');'
```

Reply:
I'm not able to see any regression 

<img width="883" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/daeaba6c-b160-4f58-aadd-1ec4f3e6f6b8">

Reply:
I agree that checking to make sure the function exists before calling it would be a good idea, since things work without calling this (we just get the console warnings).

Reply:
done -> https://github.com/woocommerce/woocommerce/pull/39027/commits/c55478e99ddc501069f2ce74409a5d7cf7c59ed3